commands/run.go

```diff
@@ -77,7 +78,8 @@ func run(ctx context.Context, cfg *config.Config, configFilepath string) {
 	connectionChecker := connectionchecker.New(&cfg.Paths)
 	v := versioner.New(&versioner.RealExecFunctions{})
 	client := client.New(cfg, supervisor, connectionChecker, v)
-	localServer := agentlocal.NewServer(cfg, supervisor, client, configFilepath)
+	ringLog := storelogs.New(500)
```

Comment:
let's move it outside the `run` function because `run` is called each time we lose connection with pmm-server